client/src/utils/gateways.ts

```diff
@@ -1,6 +1,14 @@
 import apiClient from '@/utils/http-common';
 
-export const getStudentsList = async (semester: string) => apiClient
+export const getStudentsList = async () => apiClient
```

Comment:
@JuliaZamaitat @Kaes3kuch3n Was haltet ihr davon then-chaining zu vermeiden und konsequent async-await mit try-catch zu verwenden?

Reply:
find ich gut! Würd ich aber in ein extra Issue packen :)